resources/views/home.blade.php

```diff
@@ -0,0 +1,196 @@
+@extends('layouts.main')
+
+@section('title', 'Home Page')
+
+@section('css')
+    @vite('resources/css/index.css')
+    @endsection
+
+@section('content')
+    <section>
```

Comment:
Every page that extends the layout.main all have a `<section>` as its main element inside content. It is better to put that in the main layout itself to avoid code duplication.